src/servers/CMakeLists.txt

```diff
@@ -115,6 +115,55 @@ if(${TRTIS_ENABLE_HTTP} OR ${TRTIS_ENABLE_METRICS})
   )
 endif() # TRTIS_ENABLE_HTTP || TRTIS_ENABLE_METRICS
 
+#
+# kfserving http endpoint
+#
+set(KFSERVING_HTTP_ENDPOINT_OBJECTS "")
+set(KFSERVING_HTTP_ENDPOINT_LIBRARIES "")
+
+if(${TRTIS_ENABLE_KFSERVING})
+  find_package(libevhtp CONFIG REQUIRED)
+  message(STATUS "Using libevhtp ${libevhtp_VERSION}")
+
+  set(
```

Comment:
I think you can just set HTTP_ENDPOINT_SRCS and HTTP_ENDPOINT_HDRS depending on TRTIS_ENABLE_HTTP and TRTIS_ENABLE_HTTP_V2 and everything else is the same

Reply:
Yes, I will do that